src/js/components/DataTable/StyledDataTable.js

```diff
@@ -103,12 +108,18 @@ const StyledDataTableBody = styled(TableBody)`
     overflow: auto;
   `}
 
-  &:focus {
-    ${focusStyle({ skipSvgChildren: true, forceOutline: true })}
-  }
+  ${props =>
+    props.itemFocus &&
```

Comment:
I don't think the StyledDataTableBody ever received `itemFocus` as a property.

Reply:
Resolved now, accidentally placed `itemFocus` on the Row component instead of the Table Body